.circleci/github-pages.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# This builds docs from current sceptre version and pushes it to the website with lies in separate repository
+set -e
+# show where we are on the machine
+echo "we are in:" $(pwd)
```

Comment:
could probably lose line 5 and 6

Reply:
Good catch, they were helpful during development